plugins/chruby/chruby.plugin.zsh

```diff
@@ -45,46 +43,37 @@ _source_from_omz_settings() {
     zstyle -s :omz:plugins:chruby path _chruby_path
     zstyle -s :omz:plugins:chruby auto _chruby_auto
 
-    if [[ -r ${_chruby_path} ]]; then
-        source ${_chruby_path}
+    if [[ -r "${_chruby_path}" ]]; then
+        source "${_chruby_path}"
     fi
 
-    if [[ -r ${_chruby_auto} ]]; then
-        source ${_chruby_auto}
+    if [[ -r "${_chruby_auto}" ]]; then
+        source "${_chruby_auto}"
     fi
 }
 
-_chruby_dirs() {
-    chrubydirs=($HOME/.rubies/ $PREFIX/opt/rubies)
-    for dir in chrubydirs; do
-        if [[ -d $dir ]]; then
-            RUBIES+=$dir
-        fi
-    done
-}
+function () {
+    local _chruby_homebrew_prefix="$(brew --prefix chruby) &> /dev/null"
 
-if _homebrew-installed && _chruby-from-homebrew-installed ; then
-    source $(brew --prefix chruby)/share/chruby/chruby.sh
-    source $(brew --prefix chruby)/share/chruby/auto.sh
-    _chruby_dirs
-elif [[ -r "/usr/local/share/chruby/chruby.sh" ]] ; then
-    source /usr/local/share/chruby/chruby.sh
-    source /usr/local/share/chruby/auto.sh
-    _chruby_dirs
-else
-    _source_from_omz_settings
-    _chruby_dirs
-fi
+    if _homebrew-installed && [ -r "${_chruby_homebrew_prefix}" ] ; then
```

Comment:
Let's change all these parameter expressions to get rid of the curly braces, which is more idiomatic zsh. Braces should only be used if we have to append or prepend some other string. I'll make sure we have that written in our style guide.